src/liblzma/check/crc32_fast.c

```diff
@@ -135,12 +135,12 @@ typedef uint32_t (*crc32_func_type)(
 // This resolver is shared between all three dispatch methods. It serves as
 // the ifunc resolver if ifunc is supported, otherwise it is called as a
 // regular function by the constructor or first call resolution methods.
-// The function attributes are needed for safe IFUNC resolver usage with GCC.
+// The funcion attributes are needed for safe IFUNC resolver usage with GCC.
```

Comment:
function

Reply:
this pr just reverts changes lol

Reply:
It was easier to revert the entire commit than only revert parts of it.

That entire line is removed in a subsequent commit (1efea8d8ec96676850341532970f90ec9db7d964), so the end result is the same.